src/args.rs

```diff
@@ -30,6 +30,12 @@ pub struct Args {
     #[clap(long = "all-targets")]
     /// Return dependencies for all targets. By default only the host target is matched.
     pub all_targets: bool,
+    #[clap(long = "skip-cache")]
+    /// Do not read from disk cache for Debian database results
+    pub skip_cache: bool,
+    #[clap(long = "concurrency", short = 'j', default_value = "24")]
```

Comment:
is 24 really ok? postgresql uses quite a lot of memory per connection on the server side

Reply:
It's the number that is currently used for `QUERY_THREADS`, but pretty much any other number is also fine with me if you have suggestions. :)

Reply:
Right, if it's already in use then it's hopefully not a problem :)